Aleksandr Babkin/Lesson2/game.js

```diff
@@ -0,0 +1,101 @@
+var readline = require('readline');
```

Comment:
Если копируете откуда-то код - не забывайте подгонять под общую стилистику
```suggestion
const readline = require('readline');
```